### ISSUE TITLE
BernieBNB & Maps Integration for Housing/Travel

### ISSUE THREAD

**Issue by PamelaJohn** [RichardLitt/stranded-by-trump#39]
Is it possible to integrate BernieBNB (or MarchBNB) to link immigrants with safe (vetted) housing in sanctuary cities using Rapi's mapping applications (#maps-for-change is in ideation phase)? **[will link #maps-for-change once it is created]** _It has since been pulled down but Rapi built maps.berniesanders.com which was integral to linking providers of services with seekers of services during Bernie 2016_.

Another possible consideration is integrating with some of the ride-sharing tools which were created during Bernie 2016 (BernieBus). 

Tech Reps from Movimiento Cosecha are in ProgCode Network. Bringing @rapi @henry.jacqz @arielleclynes and @samuelcole into the discussion may be invaluable.

**Comment by samuelcole** [comment]
👋 Hi!

I'm starting to think through next steps for the BNB product. I think we are interested in building a general platform where you could offer your space for housing progressive causes of all kinds. For example, you live in DC and there's a march next Saturday, or there's a crisis where suddenly more housing is needed for immigrants or refugees in specific cities.

What BNB already allows volunteers to offer housing for specific dates and allows people in need of housing to share their own email with those hosts within geographical areas. This served the campaign trail use case well because there was a built-in trust network that had low-volume hosting needs and ability across the country.

What BNB doesn't do yet:

* any sort of vetting (it's on the user's to vet, and the system doesn't give you very much to vet on)
* focus hosting needs on specific issues (would you be willing to host refugees, but not marchers)
* managing hostings/visits (with high volume dates/areas, there's a huge need to mark a hosting as 'taken' or a visit as 'hosted'. there's also been asks for a review system, which goes into the vetting point)

**Comment by PamelaJohn** [comment]
The idea of a general BNB platform for housing progressive causes is awesome. Have you considered including encryption @samuelcole? I think it's integral for supporting activists working in immigration rights given they are a triple target ATM:

-  many undocumented or documented w/soon to expire DACA or visa protections
-  vocal anti-administration critics
-  by and large successful in their outreach on social media which is under attack and frequently monitored

**Comment by samuelcole** [comment]
Yeah, I think the vetting would be a higher priority. That's not even people trying to circumvent security, it's bad actors working within the system.

**Comment by samuelcole** [comment]
Oh also @RichardLitt, don't forget about https://www.couchsurfing.com, they might also be interested in helping with housing.

**Comment by PamelaJohn** [comment]
Agreed. Out of necessity I would imagine these activists have done most of their own vetting and know who they can and cannot trust, for the most part. Of course, in this new Era concerns about vetting those who have already been vetted will be even greater. Devs might be reliant on the activists to do the vetting in that case.

In the product I've been imagining, I envision keeping the link private and having an application which acts as a digital underground railroad. A current project is underway which would make an encrypted SMS blast to a known network of individuals on Signal for emergency response.

**Comment by RichardLitt** [comment]
This all sounds great! Thank you so much for the lively discussion. I apologize for joining late - traveling at the moment. 

It looks like BNB would be good to add, but it's not in the works yet. I'll add couchsurfing - I've used it a lot, before, too. I'll see if I can contact them about helping out.

I think those are the only two actionable items I can get to in this thread - everything else is planning. I don't think that strandedbytrump.com would serve the wider BNB needs you guys are mentioning; perhaps the conversation belongs somewhere else where it can grow more?

**Comment by trl234** [comment]
This is a great idea. Is this project currently in progress?
